app/models/exchange_rate_file.rb

```diff
@@ -51,7 +51,8 @@ def filename_for(type, format, year, month)
     end
 
     def filename_for_download(type, format, year, month)
-      month_with_zero = sprintf('%02i', month)
+      # `to_i` is vital, since a number starting with 0 is interpreted as octal.
+      month_with_zero = sprintf('%02i', month.to_i(10))
```

Comment:
Isn't base 10 the default?

Reply:
Good point, yes! it is, I was thinking of removing it,
I kept it to make it clear that if we remove it,
the string might be interpreted as octal (if a string starts with '0').